WcaOnRails/app/views/competitions/_admin_index_table.html.erb

```diff
@@ -1,5 +1,5 @@
 <table class="table table-striped table-condensed admin-competitions">
-  <% toggle_tbody_message = title.include?("last") ? "hide" : "show" %>
+  <% toggle_tbody_message = title.include?("Past") ? "show" : "hide" %>
```

Comment:
What if we are on the ongoing&upcoming page, both are hidden. Does it make sense to still use it? If you open a tab, it's collapsed on search, I'm not sure if this makes life easier.